src/test/java/org/jitsi/meet/test/MalleusJitsificus.java

```diff
@@ -85,7 +87,9 @@
                 ? numParticipants
                 : Integer.parseInt(numAudioSendersStr);
 
-        int timeoutMs = 1000 * Integer.parseInt(System.getProperty(DURATION_PNAME));
+        int durationMs = 1000 * Integer.parseInt(System.getProperty(DURATION_PNAME));
+
+        int joinDelayMs = 1000 * Integer.parseInt(System.getProperty(JOIN_DELAY_PNAME));
```

Comment:
The doc above says that it is already in milliseconds?

Reply:
Whoops, you're right.